docker-compose.override.dev.yml

```diff
@@ -5,6 +5,13 @@ networks:
     name: moneywise
 
 services:
+  moneywise-ui:
+    environment: 
+      - API_BASE_URL=http://localhost:8000
```

Comment:
nginx doesn't need API_BASE_URL environment variable (or any, for that matter). Take out `environment` from this, please

Reply:
Done

Reply:
Not quite, see new comment.